src/content/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent.mdx

```diff
@@ -59,46 +59,30 @@ Ready to get started? Click one of these button to try it out.
+
 <Callout variant="important">
   If you install the agent using the [New Relic One UI](https://one.newrelic.com), the [Infrastructure status API](https://github.com/newrelic/infrastructure-agent/blob/master/docs/status_api.md) is enabled by default. 
 </Callout>
 
-## One agent, many capabilities [#overview]
-
-Our infrastructure monitoring agent collects performance and health data about the system resources and processes of the host where it's enabled (on-premises or virtualized). At the same time, it acts as a forwarder for two types of data: core services metrics, which are collected by [on-host integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations), and [logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent).
-
-If you want to collect data about core services running on your host, you need to [install the infrastructure monitoring agent](#install) first, and then [install or enable on-host integrations](/docs/integrations/host-integrations/installation/install-infrastructure-host-integrations).
-
-![Agents and New Relic](./images/agents-and-nr.png "Agents and New Relic")
-
-<figcaption>
-  Our infrastructure monitoring agent and its integrations collect data from the system and core services. It can also [forward logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent) to New Relic. Backend application metrics (APM) are collected by separate [language agents](/docs/agents). Notice how each integration and forwarder feed [different data types](/docs/using-new-relic/data/understand-data/new-relic-data-types) in the New Relic database (NRDB).
-</figcaption>
-
 ## Install the infrastructure monitoring agent [#install]
 
-If our guided install doesn't work for your setup, follow the instructions for your Linux, Windows, macOS or other setup.
-
-The infrastructure monitoring agent can currently run on many Linux distributions, Windows, and macOS. For more information on where you can run the agent, check the [compatibility and requirements](/docs/infrastructure/infrastructure-monitoring/get-started/compatibility-requirements-infrastructure-agent) page.
-
 ### Linux [#Linux]
 
-The preferred way to install the Linux agent is through the [package manager](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-agent-linux-using-package-manager) of your distribution. Select your distribution from the list for step-by-step instructions.
+To install the infrastructure agent on Linux, select your distribution from the list for step-by-step instructions. To use these installation links, you must be logged to your New Relic account.
```

Comment:
I'd remove all tiles and only mention that for detailed instructions for Linux go to: https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux/